geometry_msgs/PoseStamped[] tasks
---
bool result